fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/impl/FHIRPersistenceJDBCImpl.java

```diff
@@ -256,6 +282,8 @@ public FHIRPersistenceJDBCImpl(Properties configProps, IConnectionProvider cp, F
         // For unit tests (outside of JEE), we also need our own mechanism for handling transactions
         this.transactionAdapter = new FHIRTestTransactionAdapter(cp);
 
+        // TODO connect the transactionAdapter to our cache so that we can handle tx events in a non-JEE world
+        this.transactionDataImpl = null;
```

Comment:
this one seems useful as a backlog issue / technical debt

Reply:
Creating an issue